docs/blog/articles/2023-02-05-KServe-0.10-release.md

```diff
@@ -0,0 +1,114 @@
+# Announcing: KServe v0.10.0
+
+We are excited to announce KServe 0.10 release, in this release we have been focusing on enabling more KServe networking setup options,
+improving metrics for supported serving runtimes and increasing support coverage for [Open(aka v2) inference protocol](https://kserve.github.io/website/0.10/modelserving/data_plane/v2_protocol/) for both KServe and ModelMesh.
+
+## KServe Networking Options
+
+Istio is now optional for both `Serverless` and `RawDeployment` mode, please see the [alternative networking guide](https://kserve.github.io/website/0.10/admin/serverless/kourier_networking/) for how you can enable other ingress options supported by Knative with Serverless mode.
+For Istio users, if you want to turn on full service mesh mode to secure inference services with mutual TLS and enable the traffic policies, please read the [service mesh setup guideline](https://kserve.github.io/website/0.10/admin/serverless/servicemesh/).
+
+## KServe Telemetry for Serving Runtimes
+
+We have instrumented additional latency metrics in KServe python serving runtimes for `preprocess`, `predict` and `postprocess` handlers.
+In Serverless mode we have extended Knative queue-proxy to enable metrics aggregation for both metrics exposed in `queue-proxy` and `kserve-container`.
```

Comment:
```suggestion
In Serverless mode we have extended Knative queue-proxy to enable metrics aggregation for both metrics exposed in `queue-proxy` and `kserve-container`.  We also added support for exposing metrics from all of our serving runtimes in Serverless Mode. 
```


something like this? we have added support for each serving runtime in serverless mode.

Reply:
we also have new observability and grafana guide with open source dashboard templates.